### PR TITLE
1133 encounter created datetime

### LINKED ISSUE #1133: Add encounter created datetime field

### PR DESCRIPTION
**Pull request by clemens-msupply**
Closes #1133

### CODE REVIEW

File: server/graphql/programs/src/types/encounter.rs [hunk 1]
```diff
@@ -29,23 +29,23 @@ pub struct EncounterNode {
...
+            EncounterNodeStatus::Completed => EncounterStatus::Completed,
             EncounterNodeStatus::Cancelled => EncounterStatus::Cancelled,
         }
     }
 
     pub fn from_domain(status: &EncounterStatus) -> EncounterNodeStatus {
         match status {
             EncounterStatus::Scheduled => EncounterNodeStatus::Scheduled,
-            EncounterStatus::Done => EncounterNodeStatus::Done,
+            EncounterStatus::Completed => EncounterNodeStatus::Completed,
```
Comment (CarlosNZ) [inline server/graphql/programs/src/types/encounter.rs:48]: Good -- I changed this from "DONE" to "COMPLETED" in the schema too!

File: server/service/src/programs/schemas/patient.json [hunk 2]
```diff
@@ -266,7 +273,10 @@
           "$ref": "#/definitions/SocioEconomics"
         }
       },
-      "required": ["contactDetails", "id"],
+      "required": [
+        "contactDetails",
+        "id"
+      ],
       "type": "object"
     },
```
Comment (CarlosNZ) [inline server/service/src/programs/schemas/patient.json:281]: This all seems like whitespace changes, and I don't think it's the standard JSON formatting (the previous was), this seems like the output of the programschemas repo before re-formatting, so probably best to revert formatting?

File: server/service/src/programs/schemas/program_enrolment.json [hunk 3]
```diff
@@ -17,11 +17,18 @@
...
       "type": "object"
     },
     "ProgramEnrolmentStatus": {
-      "enum": ["ACTIVE", "OPTED_OUT", "TRANSFERRED_OUT", "PAUSED"],
+      "enum": [
+        "ACTIVE",
+        "OPTED_OUT",
+        "TRANSFERRED_OUT",
+        "PAUSED"
+      ],
```
Comment (CarlosNZ) [inline server/service/src/programs/schemas/program_enrolment.json:31]: Same here.

**Review by CarlosNZ — APPROVED** [review]
Looks good, thanks. Would you be able to re-save the .json files in this repo, so the formatting stays consistent? (The output of the programsschemas script is different to our default JSON formatting)

File: server/graphql/programs/src/types/encounter.rs (hunk 1 above)
Comment (clemens-msupply) [reply server/graphql/programs/src/types/encounter.rs:48]: thats why I changed it :)

**Pull request merged**